lib/linked_list.rb

```diff
@@ -7,39 +8,72 @@ def initialize
     @head = nil
   end
 
-  # Time complexity - ?
-  # Space complexity - ?
+  # Time complexity - O(1) where 1 is the one node created
+  # Space complexity - O(1) where 1 is the one node created in space
```

Comment:
Actually it's O(1) because the amount of time and memory doesn't change no matter how big the list is.